web-2.5/middleware/src/endpoints/write.ts

```diff
@@ -0,0 +1,19 @@
+import type { ActionResult } from '../types';
+import { builder } from 'paima-sdk/paima-concise';
+import { getActiveAddress, postConciselyEncodedData } from 'paima-sdk/paima-mw-core';
+
+async function renamePlayer(name: string): Promise<ActionResult> {
+  const userWalletAddress = getActiveAddress();
+
+  const conciseBuilder = builder.initialize();
```

Comment:
Now: `const conciseBuilder = builder.initialize();` should be 
  `const conciseBuilder = builder.initialize(undefined, ENV.CONCISE_GAME_NAME);` due to a change in the api.

Reply:
added